Godeps/_workspace/src/github.com/spf13/pflag/flag.go

```diff
@@ -566,9 +577,21 @@ func (f *FlagSet) setFlag(flag *Flag, value string, origArg string) error {
 	if len(flag.Deprecated) > 0 {
 		fmt.Fprintf(os.Stderr, "Flag --%s has been deprecated, %s\n", flag.Name, flag.Deprecated)
 	}
+	if len(flag.ShorthandDeprecated) > 0 && containsShorthand(origArg, flag.Shorthand) {
```

Comment:
I'm concerned if I called `-abcd` and `c` was deprecated that I will get the error message 4 times...